bot/exts/moderation/infraction/management.py

```diff
@@ -256,15 +257,64 @@ async def search_reason(self, ctx: Context, reason: str) -> None:
             'bot/infractions/expanded',
             params={'search': reason}
         )
+
+        formatted_infraction_count = self.format_infraction_count(len(infraction_list))
+        embed = discord.Embed(
+            title=f"Infractions matching `{reason}` ({formatted_infraction_count} total)",
+            colour=discord.Colour.orange()
+        )
+        await self.send_infraction_list(ctx, embed, infraction_list)
+
+    # endregion
+    # region: Search for infractions by given actor
+    @infraction_group.command(name="by", aliases=("b",))
+    async def search_by_actor(
+        self,
+        ctx: Context,
+        actor: t.Union[discord.Member, t.Literal["m", "me"]],
+        oldest_first: bool = False
+    ) -> None:
+        """
+        Search for infractions made by `actor`.
+
+        Use "m" or "me" as the `actor` to get infractions by author.
+
+        Use "1" for `oldest_first` to send oldest infractions first.
+        """
+        if isinstance(actor, str):
+            actor = ctx.author
+
+        infraction_list = await self.bot.api_client.get(
+            'bot/infractions/expanded',
+            params={
+                'actor__id': str(actor.id),
+                'ordering': f'{"-"[oldest_first:]}inserted_at'  # `'inserted_at'` makes api return oldest first
```

Comment:
This is smart, but it's also very hard to read what's going on here. Could you move this above the `get` call, something like:

```py
if oldest_first:
    ordering = '-inserted_at'  # Descending order
else:
    ordering = 'inserted_at'  # Ascending order
```